src/Api.Rest.Dtos/Data/InspectionPlanDtoBase.cs

```diff
@@ -52,37 +53,52 @@ public abstract class InspectionPlanDtoBase : IAttributeItemDto
 		/// Gets or sets the uuid of this inspection plan entity. The uuid is always constant, even if
 		/// this entity is renamed.
 		/// </summary>
-		[JsonProperty( "uuid" )]
+		[Newtonsoft.Json.JsonProperty( "uuid" )]
+		[JsonPropertyName( "uuid" )]
 		public Guid Uuid { get; set; }
 
 		/// <summary>
 		/// Gets or sets the comment of this inspection plan entity.
 		/// </summary>
-		[JsonProperty( "comment" )]
+		[Newtonsoft.Json.JsonProperty( "comment" )]
+		[JsonPropertyName( "comment" )]
 		public string Comment { get; set; }
 
 		/// <summary>
 		/// Gets or sets the path of this inspection plan entity.
 		/// </summary>
+		[Newtonsoft.Json.JsonIgnore]
 		[JsonIgnore]
 		public PathInformationDto Path
 		{
 			[NotNull] get => _Path ?? PathInformationDto.Root;
 			set => _Path = value;
 		}
 
+		/// <summary>
+		/// Gets or sets the path of this inspection plan entity.
+		/// </summary>
+		[JsonInclude, JsonPropertyName( "path" ), JsonConverter( typeof( JsonPathInformationConverter ) )]

```

Comment:
Can you explain why this is needed? Is there a difference on how Newtonsoft and System.Text work with the private Path property?

Reply:
System.Text.Json is more restrictive on non-public fields and property accessors.

- A non-public field or property cannot be serialized, even if annotated.
- A public field can be serialized, but has to be annotated with `JsonInclude`.
- A public property can be serialized, even if any accessor is non-public, but has to be annotated with `JsonInclude`.

See https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-immutability?pivots=dotnet-6-0#non-public-property-accessors

The `Path` property cannot be used for serialization, because it contains logic to never return `null`. Adding the `PathCore` property is the workaround, I decided to make the get accessor non-public to prevent the most likely use case.

Do you have any better idea? Maybe a better name for the property: `InternalPath`, `RawPath`, ...

Reply:
As we discussed today: it seems ok to have System.Text.Json use the public property Path. So Newtonsoft still uses the private property _Path and ignores Path, and System.Text.Json works the other way around, ignoring _Path and using Path.

Reply:
Done, see 8930d1e057801f9e21f0d4d5e384cf702a63a32b